assets/js/mixins/formatter.js

```diff
@@ -101,11 +101,13 @@ export default {
     fmtPricePerKWh: function (amout = 0, currency = "EUR") {
       let unit = currency;
       let value = amout;
+      let maximumFractionDigits = 3;
       if (["EUR", "USD"].includes(currency)) {
         value *= 100;
         unit = "ct";
```

Comment:
Minor Unit vom Backend wäre nett, können wir nachreichen: https://stackoverflow.com/questions/70631321/how-can-i-get-the-minor-currency-unit

Reply:
Ja können wir bei Gelegenheit mal machen. [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat) kann aktuell leider kein Minor Currency. Generell gibts sogar ne native Unterstützung für compound units (currency-per-kWh), allerdings haben es die Energieeinheiten [nicht in die aktuelle Spec](https://tc39.es/proposal-unified-intl-numberformat/section6/locales-currencies-tz_proposed_out.html#sec-issanctionedsimpleunitidentifier) geschafft :( Aber gut, das bisschen manuelle Formatierung wird uns noch nicht umbringen :D